### DISCUSSION TITLE
darwin-monterey-x86_64 did not provide module clingo

### DISCUSSION THREAD

**Discussion by HealthyPear** [spack/spack#41815] [Q&A]
I am on macos Big Sur 11.7.10

my `uname -a` output is the following

> Darwin StargateMacBookPro.station 20.6.0 Darwin Kernel Version 20.6.0: Thu Jul  6 22:12:47 PDT 2023; root:xnu-7195.141.49.702.12~1/RELEASE_X86_64 x86_64

whenever I try to install any package I see this warning, but I don't really know how much it is a problem

`==> Warning: Spec clingo-bootstrap@=spack%apple-clang@=14.0.0~docs+ipo+optimized+python build_system=cmake build_type=Release generator=make arch=darwin-monterey-x86_64 ^[deptypes=build] bison@=3.8.2%apple-clang@=14.0.0 build_system=autotools arch=darwin-monterey-x86_64 ^[deptypes=build] cmake@=3.27.6%apple-clang@=14.0.0~doc+ncurses+ownlibs build_system=generic build_type=Release arch=darwin-monterey-x86_64 ^[deptypes=build] gmake@=4.4.1%apple-clang@=14.0.0~guile build_system=autotools arch=darwin-monterey-x86_64 ^[deptypes=build,link,run] python@=3.11%apple-clang@=14.0.0+bz2+crypt+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter+uuid+zlib build_system=generic patches=ebdca64,f2fd060 arch=darwin-monterey-x86_64 ^[deptypes=build] re2c@=2.2%apple-clang@=14.0.0 build_system=generic arch=darwin-monterey-x86_64 did not provide module clingo`

**Comment by tgamblin** [comment]
It's saying that it tried to load that install, but wasn't able to import `clingo` (which we use for the solver) after doing so.

I'm not sure *why* it can't get `clingo` from there, though, since it *is* a `clingo` install.  @alalazo may be able to say more

**Reply by alalazo** [reply]
Hard to say what's happening without more information. @HealthyPear How did you bootstrap Spack? Can you post the output of:
```
spack debug report
```
for the Spack instance where this happens?

**Reply by HealthyPear** [reply]
* **Spack:** 0.22.0.dev0 (d3c1f7a872a4097b74085b7275c338a1b52d96ae)
* **Python:** 3.11.7
* **Platform:** darwin-bigsur-broadwell
* **Concretizer:** clingo

**Reply by HealthyPear** [reply]
I don't know if this is the problem, but I have _bigsur_ and it seems that he's trying to look for _monterey_

**Reply by alalazo** [reply]
Can you post:
```console
$ spack -b find
```
?

**Reply by HealthyPear** [reply]
```
-- darwin-bigsur-x86_64 / apple-clang@13.0.0 --------------------
bison@3.8.2  clingo-bootstrap@spack  cmake@3.24.2  python@3.11

-- darwin-monterey-x86_64 / apple-clang@14.0.0 ------------------
bison@3.8.2  clingo-bootstrap@spack  cmake@3.27.6  python@3.11
==> 8 installed packages
```
